src/api/backend.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+import { ethers } from 'ethers';
+
+export const getAddressLLM = async (walletAddress) => {
+  const ssoAccessTokenHash = sessionStorage.getItem('ssoAccessTokenHash');
+
+  const api2 = axios.create({
+    baseURL: process.env.REACT_APP_API2,
+    withCredentials: true,
+  });
+  api2.defaults.headers.common['X-Token'] = ssoAccessTokenHash;
+
+  const { data } = await api2.get('/users', {
+    params: {
+      blockchainAddress: walletAddress,
+      limit: 1,
+    },
+  });
+
+  return ethers.utils.parseUnits(data[0].merits.toFixed(12), 12).toString();
```

Comment:
Why `.toString()`? It's better to keep it as BigNumber, so that we can use it with polkadot.js apps later.